datadog_checks_dev/datadog_checks/dev/tooling/templates/configuration/instances/openmetrics.yaml

```diff
@@ -25,7 +25,7 @@
     items:
       type: string
 - name: prometheus_metrics_prefix
-  description: <PREFIX> for exposed Prometheus metrics.
+  description: Removes a given <PREFIX> from exposed Prometheus metrics.
   value:
     type: string
     example: <PREFIX>_
```

Comment:
What is the purpose of `_` after `<PREFIX>`?

Reply:
I guess because services adding a prefix usually add an extra `_` between the prefix and the rest of the metric name for readability?

Reply:
Gotcha, so it would remove the extra `_` that is also included.